src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -706,7 +772,7 @@ private void refreshBeansList() throws IOException {
         }
 
         this.beans = (this.beans.isEmpty()) ? connection.queryNames(null) : this.beans;
-        this.lastRefreshTime = System.currentTimeMillis();
+        this.lastRefreshTime = System.currentTimeMillis(); 
```

Comment:
whitespace at the end of the line, look up how to configure VSCode to automatically trim trailing whitespace